data/virt_autotest/fetch_logs_from_guest.sh

```diff
@@ -55,7 +55,7 @@ function fetch_logs_from_guest_via_ssh() {
 
         local ret_result=128
         local retry_times=0
-        while [[ ${retry_times} -lt 3 ]] && [[ ${ret_result} -ne 0 ]];
+        while [[ ${retry_times} -lt 2 ]] && [[ ${ret_result} -ne 0 ]];
```

Comment:
Why changed to ```2```

Reply:
We should find a good balance between fetching/uploading logs success ratio and the needed time. It is not rare to see that failing jobs take several hours to finish/stop job in osd any my own alp tests. We need to step back a little bit to let fail sooner to pursue higher test failure efficiency.

Reply:
You can lower the number. Actually I am not obsessed with "retrying too much". Hope you can find "the new balance".